src/main/scala/cromwell/engine/ExecutionStatusStore.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.engine
```

Comment:
This is more a note to myself (via miguel who can remind me :)) but I suspect all these stores will have some commonality  (e.g. a collection[T] of the values, retrieving status, etc). That should all be pulled into a trait